.travis.yml

```diff
@@ -44,22 +45,37 @@ before_install:
 - export PATH=$PATH:/home/travis/gopath/src/github.com/radanalyticsio/oshinko-cli/origin/
 - echo $PATH
 # below cmd is important to get oc working in ubuntu
-- if [ "$TO_TEST" = "e2e" ]; then sudo docker run -v /:/rootfs -ti --rm --entrypoint=/bin/bash --privileged openshift/origin:v1.5.1 -c "mv /rootfs/bin/findmnt /rootfs/bin/findmnt.backup" ; fi
-- if [ "$TO_TEST" = "e2e" ]; then sudo oc cluster up --host-config-dir=/home/travis/gopath/src/github.com/radanalyticsio/oshinko-cli/origin ; fi
-- if [ "$TO_TEST" = "e2e" ]; then sudo chmod -R a+rwX /home/travis/.kube ; fi
-- if [ "$TO_TEST" = "e2e" ]; then sudo ls -l /home/travis/gopath/src/github.com/radanalyticsio/oshinko-cli/origin ; fi
-# find IP:PORT of openshift
-- if [ "$TO_TEST" = "e2e" ]; then IPSTR=`sudo oc status |grep server` ; fi
-- if [ "$TO_TEST" = "e2e" ]; then echo $IPSTR ; fi
-- if [ "$TO_TEST" = "e2e" ]; then IP=${IPSTR##*/} ; fi
-- if [ "$TO_TEST" = "e2e" ]; then echo ${IP} ; fi
-- if [ "$TO_TEST" = "e2e" ]; then IPSTR=`sudo oc login -u system:admin` ; fi
-- if [ "$TO_TEST" = "e2e" ]; then echo $IPSTR ; fi
+- sudo docker run -v /:/rootfs -ti --rm --entrypoint=/bin/bash --privileged openshift/origin:v1.5.1 -c "mv /rootfs/bin/findmnt /rootfs/bin/findmnt.backup"
```

Comment:
i wonder if we want to parameterize the openshift version at some point?

Reply:
ack. I have an idea to move the install/startup to a bash script in oshinko-release that gets sucked in and executed by travis.  Parameterization can be made part of that. At that point, the local travis.yml becomes trivial and we've got the same setup in all repos.